packages/datagateway-dataview/cypress/integration/table/isis/facilityCycles.spec.ts

```diff
@@ -24,13 +24,14 @@ describe('ISIS - FacilityCycles Table', () => {
     cy.get('[aria-rowcount="75"]').should('exist');
   });
 
-  it('should be able to resize a column', () => {
+  // TODO: Having three columns does not work since the width not calulated correctly.
+  it.skip('should be able to resize a column', () => {
     let columnWidth = 0;
 
     cy.window()
       .then((window) => {
         const windowWidth = window.innerWidth;
-        columnWidth = windowWidth / 4;
+        columnWidth = windowWidth / 3;
```

Comment:
It's interesting because the columnWidth variable comes to 1000 / 3, which is exactly what the width of the columns should be and yet it determines them to be 327.666...

Reply:
@sam-glendenning the issue with this is the scrollbar being present when the data is loaded in the table. The CSS width for the table however is `1000px`. One possible way to get this to work might be to stop the data from being loaded for the specific e2e test and then check the width is what we expect it to be.

Reply:
As @sam-glendenning has done in the latest commit, we now use the filtering box to enter a filter term that does not match the data to ensure no data is loaded. As a result, a scrollbar does not appear. To get around the issue of the width values not matching exactly, we round it to 1 decimal place and then compare them with the column width.